kart/raster/metadata_util.py

```diff
@@ -18,12 +19,24 @@
 CATEGORIES_PATTERN = re.compile(r"band/band-(.*)-categories\.json")
 
 
-def rewrite_and_merge_metadata(tile_metadata_list):
+class RewriteMetadata(IntEnum):
```

Comment:
since you're using these with bitwise operators maybe it should be an `IntFlag`

Reply:
Done